pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -50,6 +48,16 @@ const (
 	Name CloudStackResourceRefType = "Name"
 )
 
+// Zone is an organizational construct typically used to represent a single datacenter, and all its physical and virtual resources exist inside that zone. It can either be specified as a UUID or name
+type Zone struct {
```

Comment:
Should we find a different name for this struct?
It seems with this like a zone contains a zone ref and a network
That makes me think that `Zone` should be something else
What could be the construct that groups a zone and a network?
```go
type SomethingElse struct {
  Zone CloudStackResourceRef
  Network CloudStackResourceRef
}
```

Reply:
refactored.